tests/testthat/test-old_new.R

```diff
@@ -1,23 +1,85 @@
+# The purpose of this test is to ensure that with minor package development there is no
```

Comment:
FYI @jianqiuz the first round of old new testing, I compared results returned by the package functions MEND, MEND2, MEND2_ECA, and so on with the csv files returned with your R scripts & the difference between the results had a MSE of <= 1e-24

Reply:
Very nice!